src/translate.zig

```diff
@@ -113,20 +113,46 @@ pub fn buffer_from_object(
         return throw(env, key ++ " must be defined");
     }
 
+    return buffer_from_value(env, property, key);
+}
+
+pub fn buffer_from_value(
+    env: c.napi_env,
+    value: c.napi_value,
+    comptime key: [*:0]const u8,
+) ![] u8 {
     var is_buffer: bool = undefined;
-    assert(c.napi_is_buffer(env, property, &is_buffer) == .napi_ok);
+    assert(c.napi_is_buffer(env, value, &is_buffer) == .napi_ok);
 
     if (!is_buffer) return throw(env, key ++ " must be a buffer");
 
     var data: ?*c_void = null;
     var data_length: usize = undefined;
-    assert(c.napi_get_buffer_info(env, property, &data, &data_length) == .napi_ok);
+    assert(c.napi_get_buffer_info(env, value, &data, &data_length) == .napi_ok);
 
     if (data_length < 1) return throw(env, key ++ " must not be empty");
 
     return @ptrCast([*]u8, data.?)[0..data_length];
 }
 
+pub fn copy_buffer_of_length_from_object(env: c.napi_env, object: c.napi_value, comptime length: u8, comptime key: [:0]const u8) ![length]u8 {
```

Comment:
Because we return the actual array, and not a slice or pointer, the function signature itself shows that the resulting array's memory is owned by the call site.

What do you think of calling this `bytes_from_object()` and then `buffer_from_object()` and `buffer_from_value()` can be `slice_from_object()` and `slice_from_value` respectively?